docs/pages/database-access/guides/cockroachdb-self-hosted.mdx

```diff
@@ -91,11 +91,21 @@ $ tctl auth sign \
     --ttl=2190h
 ```
 
-The command will produce 3 files: `ca.crt` with Teleport's certificate authority
-and `node.crt` / `node.key` with the node's certificate and key. Do not rename
-them as this is how CockroachDB expects them to be named. See [Node key and certificates](https://www.cockroachlabs.com/docs/v21.1/create-security-certificates-custom-ca#node-key-and-certificates)
+The command will produce 4 files:
+- `ca.crt` with Teleport's `db` certificate authority
+- `ca-client.crt` with Teleport's `db_client` certificate authority
+- `node.crt` / `node.key` with the node's certificate and key.
+
+Do not rename these files as this is how CockroachDB expects them to be named.
+See [Node key and certificates](https://www.cockroachlabs.com/docs/v21.1/create-security-certificates-custom-ca#node-key-and-certificates)
 for details.
 
+Teleport uses a split CA architecture for database access, and requires that
```

Comment:
I would make this paragraph a prerequisite for the guide so users aren't surprised by the need to step away from the Teleport docs and understand another docs page. Once the reader has all of the external steps taken care of, they can fly through this page by copying/pasting commands etc.

Reply:
done f87e7d7